lib/super_digit.rb

```diff
@@ -1,11 +1,32 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
 def super_digit(n)
+  super_digit_helper({}, n)
    
 end
-  
+
+def super_digit_helper(solutions, n)
```

Comment:
Interesting approach.  Nice work.  This would work more often if you had repeated uses of `super_digit`